netaddr.go

```diff
@@ -971,15 +971,23 @@ func MustParseIPPort(s string) IPPort {
 }
 
 // IsZero reports whether p is its zero value.
+// All ports are valid, including zero.
```

Comment:
This comment should go in Valid's docstring or the type's docstring, not IsZero's.

Reply:
Oops. Fixed, thanks.